### DISCUSSION TITLE
Welcome to Airup Discussions!

### DISCUSSION THREAD

**Discussion by sisungo** [sisungo/airup#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hetaodo** [comment]
Hope you can make what you want! Feel the real charm of programming!

**Comment by sisungo** [comment]
Should Airup's IPC protocol switch to use a binary serialization format, instead of JSON?
